lib/sisimai/bite/email/aol.rb

```diff
@@ -0,0 +1,208 @@
+
+            if connvalues == connheader.keys.size
+              # Final-Recipient: rfc822; kijitora@example.co.jp
+              # Original-Recipient: rfc822;kijitora@example.co.jp
+              # Action: failed
+              # Status: 5.2.2
+              # Remote-MTA: dns; mx.example.co.jp
+              # Diagnostic-Code: smtp; 550 5.2.2 <kijitora@example.co.jp>... Mailbox Full
+              v = dscontents[-1]
+
+              if cv = e.match(/\A[Ff]inal-[Rr]ecipient:[ ]*(?:RFC|rfc)822;[ ]*([^ ]+)\z/)
+                # Final-Recipient: RFC822; userunknown@example.jp
+                if v['recipient']
+                  # There are multiple recipient addresses in the message body.
+                  dscontents << Sisimai::Bite.DELIVERYSTATUS
+                  v = dscontents[-1]
+                end
+                v['recipient'] = cv[1]
+                recipients += 1
+
+              elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+                # Action: failed
+                v['action'] = cv[1].downcase
+
+              elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+                # Status:5.2.0
+                v['status'] = cv[1]
+
+              elsif cv = e.match(/\A[Rr]emote-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Remote-MTA: DNS; mx.example.jp
+                v['rhost'] = cv[1].downcase
+
+              else
+                # Get error message
+                if cv = e.match(/\A[Dd]iagnostic-[Cc]ode:[ ]*(.+?);[ ]*(.+)\z/)
+                  # Diagnostic-Code: SMTP; 550 5.1.1 <userunknown@example.jp>... User Unknown
+                  v['spec'] = cv[1].upcase
+                  v['diagnosis'] = cv[2]
+
+                elsif p =~ /\A[Dd]iagnostic-[Cc]ode:[ ]*/ && cv = e.match(/\A[ \t]+(.+)\z/)
+                  # Continued line of the value of Diagnostic-Code header
+                  v['diagnosis'] ||= ''
+                  v['diagnosis']  += ' ' + cv[1]
+                  havepassed[-1] = 'Diagnostic-Code: ' + e
+                end
+              end
+
+            else
+              # Content-Type: message/delivery-status
+              # Content-Transfer-Encoding: 7bit
+              #
+              # Reporting-MTA: dns; omr-m5.mx.aol.com
+              # X-Outbound-Mail-Relay-Queue-ID: CCBA43800007F
+              # X-Outbound-Mail-Relay-Sender: rfc822; shironeko@aol.example.jp
+              # Arrival-Date: Fri, 21 Nov 2014 17:14:34 -0500 (EST)
+              if cv = e.match(/\A[Rr]eporting-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Reporting-MTA: dns; mx.example.jp
+                next if connheader['lhost'].size > 0
+                connheader['lhost'] = cv[1].downcase
+                connvalues += 1
+
+              elsif cv = e.match(/\A[Aa]rrival-[Dd]ate:[ ]*(.+)\z/)
+                # Arrival-Date: Wed, 29 Apr 2009 16:03:18 +0900
+                next if connheader['date'].size > 0
+                connheader['date'] = cv[1]
+                connvalues += 1
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->